### ISSUE TITLE
SVT

### ISSUE THREAD

**Issue by ttkxyy** [Backdrop9019/DA_project#1]
Hi, I’m sorry to disturb you here. May I ask if you have successfully reproduced the pre-training model of SVT?

**Comment by Backdrop9019** [comment]
Yes, the Author does not provide the slow-fast inference, so the results are not exactly the same, but I obtained similar results.

**Comment by ttkxyy** [comment]
Wow, you’re amazing. Can I ask you about some parameters during the pre-training stage? I set the learning rate as lr=5.0e-4*(32/256), epochs=20, weight_decay =0.04—>0.1, rand_fr=True, and other parameters are set according to the open source code. I used slow_fast inference and got 92.01 on the UCF-101 dataset and only 64.17 on the HMDB dataset. However, with the weights provided by the author, I can reach 94.23 and 68.18 respectively. I wonder if I set some parameters incorrectly or missed something during the pre-training stage?

**Comment by Backdrop9019** [comment]
Did you set the global view in rand_fr to 8 and 16? From what I remember, I think the default setting in the code was 4 and 8.
